ruby/red-arrow/test/values/test-basic-arrays.rb

```diff
@@ -286,6 +286,21 @@ def test_month_interval
     target = build(Arrow::MonthIntervalArray.new(values))
     assert_equal(values, target.values)
   end
+
+  def test_day_time_interval
+    values = [
+      {day: 1, millisecond: 100},
+      nil,
+      {day: 2, millisecond: 300},
+    ]
+    inputs = [
+      Arrow::DayMillisecond.new(1, 100),
+      nil,
+      Arrow::DayMillisecond.new(2, 300),
+    ]
+    target = build(Arrow::DayTimeIntervalArray.new(inputs))
```

Comment:
Ah, we should add support for `Arrow::DayTimeIntervalArray.new([{day: 1, millisecond: 100}, nil, {...}])`.
Could you open an issue for this improvement?

Reply:
OK!
I opened it!
https://issues.apache.org/jira/browse/ARROW-15918 

And I challenge to support this!!